enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/types/embed.ts

```diff
@@ -23,7 +24,7 @@ export type SdkIframeEmbedMessage =
   | {
       type: "metabase.embed.submitSessionToken";
       data: {
-        authMethod: SdkIframeAuthMethod;
+        authMethod: MetabaseAuthMethod;
```

Comment:
We already have a `MetabaseAuthMethod` type, so `SdkIframeAuthMethod` is redundant.